setup.py

```diff
@@ -4,8 +4,7 @@
 
 
 INSTALL_REQUIREMENTS = [
-    "Django>=1.11,<2.2",
-    "django-cms>=3.5.0",
```

Comment:
@jonathan-s is djangocms not required for testing locally?

Reply:
Apologies, deleted the entire line by mistake. If you check the most recent diff it has been added in.